neqo-bin/src/client/http3.rs

```diff
@@ -106,7 +106,7 @@ pub(crate) fn create_client(
 }
 
 impl TryFrom<Http3State> for CloseState {
-    type Error = ConnectionError;
+    type Error = CloseReason;
```

Comment:
```suggestion
    type Reason = CloseReason;
```
?

Reply:
`TryFrom` is defined in the standard library. So is the Associated type `Error` of the `TryFrom` trait. 

https://doc.rust-lang.org/std/convert/trait.TryFrom.html

Thus one can not change the name of the associated type.

Reply:
Ah got it.